### ISSUE TITLE
Voiding Payment

### ISSUE THREAD

**Issue by fbridger** [better-reports/QuickBooksSharp#8]
It seems that the DataService is not able to void a payment.

```csharp
var dataService = new QuickBooksSharp.DataService("AccessToken", "RealmId", useSandbox: true);

// https://developer.intuit.com/app/developer/qbo/docs/api/accounting/most-commonly-used/payment#void-a-payment
// Use a sparse update operation with include=void to void an existing Payment object;
// include a minimum of Payment.Id and Payment.SyncToken.
// The transaction remains active but all amounts and quantities are zeroed
// and the string, Voided, is injected into Payment.PrivateNote, prepended to existing text if present.
// If funds for the payment have been deposited, you must delete the associated deposit object before voiding the payment object.
var payment = new Payment()
{
    Id = paymentId,
    SyncToken = syncToken,
    status = EntityStatusEnum.Voided,
    sparse = true,
    PrivateNote = "Voided by QuickBooksSharp"
};

var result = await dataService.PostAsync(payment);
```

Any idea if we can actually do this?

**Comment by fbridger** [comment]
@clement911 I've created a PR that fixes this problem. Please check it out.

**Comment by clement911** [comment]
Merged.

**Comment by clement911** [comment]
Published in 1.8.1
